### ISSUE TITLE
Erroring Scripts exit with Code 0 instead of 1

### ISSUE THREAD

**Issue by willrstern** [ericclemmons/per-env#4]
package.json
```
"scripts": {
  "foo": "per-env",
  "foo:development": "blahblah" //will error command not found
}
```
terminal
```
npm run foo
echo $? #0
npm run foo:development
echo $? #1
```

**Comment by ericclemmons** [comment]
AGH!!

**Comment by ericclemmons** [comment]
Fixed via d80ce4cd2c1cc0520f0a0d2f23ac08665e774f0f & in `v1.0.2`.
